ws-nextjs-app/pages/[service]/send/[id]/FormField/styles.ts

```diff
@@ -20,11 +20,14 @@ export default {
         },
       },
     }),
-  fieldLabel: () =>
-    css({
-      display: 'inline-block',
-      marginBottom: `${pixelsToRem(6)}rem`,
-    }),
+  fieldLabel:
+    (useErrorTheme: boolean) =>
+    ({ palette }: Theme) =>
+      css({
+        display: 'inline-block',
+        marginBottom: `${pixelsToRem(6)}rem`,
+        color: useErrorTheme ? palette.ERROR_CORE : palette.GREY_10,
```

Comment:
Should we be using ternary logic within style files? Would that create the problem we had when we used to use [CSS in JS](https://latest--5d28eb3fe163f6002046d6fa.chromatic.com/?path=/docs/coding-standards-styles--readme#css-in-js-be-aware-that-passing-props-to-styled-components-will-generate-a-new-class-for-different-arguments)?